templates/category/edit.html.twig

```diff
@@ -1,15 +1,52 @@
 {% extends 'base.html.twig' %}
 
```

Comment:
redagavimui ir kūrimui galima perpanaudoti tą patį template ir nereik atskiro, jei pastebėsi be to, kad skirias title jie tokie patys.